packages/xod-client/src/editor/components/inspectorWidgets/pinWidgets/PinWidget.jsx

```diff
@@ -18,12 +18,19 @@ const isNonBindableOutput = R.both(
   R.complement(R.prop('isBindable'))
 );
 
-const isBindingForbidden = R.either(isLinkedInput, isNonBindableOutput);
+const isGenericPin = R.pipe(R.prop('dataType'), isGenericPinType);
```

Comment:
Don’t we have `XP.getPinType`? I think we should.

Reply:
We have it, but it looks at `type` prop.